src/apify/_actor.py

```diff
@@ -732,18 +733,35 @@ async def start(
         else:
             serialized_webhooks = None
 
+        if timeout == 'RemainingTime':
+            actor_start_timeout = self._get_remaining_time()
+        elif isinstance(timeout, str):
+            raise ValueError(
+                f'`timeout` can be `None`, `RemainingTime` literal or `timedelta` instance, but is {timeout=}'
+            )
+        else:
+            actor_start_timeout = timeout
```

Comment:
Maybe swap the 2nd and 3rd branch would be better?

```suggestion
        if timeout == 'RemainingTime':
            actor_start_timeout = self._get_remaining_time()
        elif timeout is None:
            actor_start_timeout = None
        elif isinstance(timeout, timedelta):
            actor_start_timeout = timeout
        else:
            raise ValueError(
                f'Invalid timeout {timeout!r}: expected `None`, `"RemainingTime"`, or a `timedelta`.'
            )
```